### DISCUSSION TITLE
How To Access Gltf Model vertices?

### DISCUSSION THREAD

**Discussion by tapmyads-dev1** [google/filament#6340] [General]
I am trying to show point on gltf face model vertices position .
is there any way to create this ?
![augmented-faces-468-point-face-mesh](https://user-images.githubusercontent.com/57178570/205053887-aa53a16c-8b1b-47cc-80aa-080713cc2e92.png)

**Comment by tapmyads-dev1** [comment]
is there any resolution on this? since this discussion has been closed

**Comment by johnny761127** [comment]
is there any higher resolution on this picture, thanks
